.github/workflows/build.yaml

```
@@ -0,0 +1,68 @@
+name: Build Stage1 packages (no push)
+
+on:
+  schedule:
+    - cron: '0 0 * * 1' # Every Monday at midnight
+
+  # TODO: remove this before merging
+  pull_request:
+    branches:
+      - main
+
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Build Wolfi Stage1
+    if: github.repository == 'wolfi-dev/bootstrap-stage1'
+
+    strategy:
+      matrix:
+        arch: [ "x86_64", "aarch64" ]
+      fail-fast: false
+
+    runs-on: wolfi-os-builder-${{ matrix.arch }}
```

```suggestion
    runs-on: wolfi-builder-${{ matrix.arch }}
```

the os variants are capped at 1, so the suggestion ensures these don’t lock lock wolfi/os releases